ifupdown2/lib/nlcache.py

```diff
@@ -2966,6 +2966,10 @@ def link_add_vxlan_with_info_data(self, ifname, info_data):
                     cmd.append("nolearning")
                 info_data[nlpacket.Link.IFLA_VXLAN_LEARNING] = int(learning)
 
+                if not udp_csum:
+                    cmd.append("noudpcsum")
+                info_data[nlpacket.Link.IFLA_VXLAN_UDP_CSUM] = int(learning)
+
```

Comment:
I got really confused here because github didn't show any context haha! 
I didn't realize I left those debug comments there! 

Maybe simply remove this, or to avoid confusion change the `int(learning)` to `int(udp_csum)`